src/Microsoft.Data.SqlClient/tests/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj

```diff
@@ -63,6 +59,7 @@
     <PackageReference Include="microsoft.dotnet.buildtools.testsuite" Version="2.2.0-preview1-03110-01" />
     <PackageReference Include="System.Linq.Expressions" Version="4.3.0" />
     <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.5.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.1'">
     <PackageReference Include="System.Data.Odbc" Version="4.5.0" />
```

Comment:
I was able to build .net core 3.0 after adding this condition:
```
<ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0'">
    <PackageReference Include="System.Data.Odbc" Version="4.6.0" />
  </ItemGroup>
```

Reply:
Seems like building and running tests with netcoreapp3.0 also needs some fixes. Leaving this change as it needs to be addressed specifically.